frame/election-provider-support/src/onchain.rs

```diff
@@ -103,6 +110,26 @@ impl<T: Config> ElectionProvider for OnChainSequentialPhragmen<T> {
 	}
 }
 
+impl<T: Config> ElectionProvider for OnChainSequentialPhragmen<T> {
+	type AccountId = T::AccountId;
+	type BlockNumber = T::BlockNumber;
+	type Error = Error;
+	type DataProvider = T::DataProvider;
+
+	fn elect() -> Result<Supports<T::AccountId>, Self::Error> {
+		Self::elect_with(None, None)
+	}
+}
+
+impl<T: Config> InstantElectionProvider for OnChainSequentialPhragmen<T> {
+	fn instant_elect(
+		maybe_max_voters: Option<usize>,
+		maybe_max_targets: Option<usize>,
```

Comment:
```suggestion
		maybe_max_voters: Option<usize>,
		maybe_max_targets: Option<usize>,
```

Usually i would suggest we keep runtime traits and apis to use u32 explicitly.

Reply:
ahh sorry hombre have to revert this. the election bounds are all usize all the way down. I do, however, as you suggested, keep them u32, when in storage or in dispatch arguments.